updates/9.sql

```diff
@@ -19,6 +19,10 @@
 #     - If they are in a n:m table, they should be NOT NULL without default
 #     - If they are in a column, they should be NOT NULL with default -1 or the corresponding default value matching "-" or "--" (NOT "?"!)
 #   - So far we will NOT set any key to use ON DELETE CASCADE or ON DELETE UPDATE. This might be done at a later point if needed.
+
+#Otherwise an error will appear. Will be restored at the end of the script
+SET FOREIGN_KEY_CHECKS=0;
```

Comment:
There has been no error to me with the old migration. As we detected in a separate PR, simply using this might screw up foreign references. Did you test this thoroughly as well?

Reply:
Yes we have tested the script 9.sql. There are many statements that add FK Constraints.
Without the FOREIGN_KEY_CHECKS=0 statement the script runs in an error

Reply:
Can you please post an example for one of these errors?